frontends/web/src/routes/market/swap/components/swap-service-selector.module.css

```
@@ -0,0 +1,105 @@
+    border: 1px solid var(--background-quaternary);
+    border-radius: var(--radius-full);
+    display: inline-flex;
+    font-size: var(--size-smaller);
+    height: calc(var(--space-half) + var(--space-eight));
+    justify-content: center;
+    margin-left: calc((var(--space-three-quarter) / 2) * -1);
+    min-width: calc(var(--space-half) + var(--space-eight));
+    padding: 0 var(--space-eight);
+}
+
+.select :global(.react-select__menu) {
```

Stylelint (17.7.0)

[error] Unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)